src/lib.rs

```diff
@@ -625,11 +625,12 @@ impl Config {
                     "powerpc64-unknown-linux-gnu" => Some("powerpc-linux-gnu"),
                     "powerpc64le-unknown-linux-gnu" => Some("powerpc64le-linux-gnu"),
                     "mips-unknown-linux-gnu" => Some("mips-linux-gnu"),
+                    "mipsel-unknown-linux-gnu" => Some("mipsel-linux-gnu"),
                     "i686-pc-windows-gnu" => Some("i686-w64-mingw32"),
                     "x86_64-pc-windows-gnu" => Some("x86_64-w64-mingw32"),
                     "x86_64-unknown-linux-musl" => Some("musl"),
                     "x86_64-rumprun-netbsd" => Some("x86_64-rumprun-netbsd"),
-                    _ => None,
+                    _ => fail(&format!("Unknown target prefix `{}`", &target[..])),
```

Comment:
This should stick around as `None` for now as it mirrors the behavior of many other standard cross-compilation setups. It's unfortunate, but it could always be the case that a cross compile with gcc actually works!